docs/FIRRTLAnnotations.md

```diff
@@ -735,4 +735,14 @@ Example:
   "source":"~GCTMemTap|GCTMemTap>mem"
 }
 ```
+#### Design Under Test
 
+| Property   | Type   | Description                                   |
+| ---------- | ------ | -------------                                 |
+| class      | string | `sifive.enterprise.firrtl.MarkDUTAnnotation`  |
+| target     | string | Reference target                              |
+
+Marks what is the DUT (and not the testbench). 
+This annotation can be lowered to the attribute 
+`DesignUnderTest = [true]` 
+to indicate the module which is the DUT.
```

Comment:
Nit: formatting here seems too tight? I think this can line wrap at 80 and use one fewer line?